src/Utils/messages-media.ts

```diff
@@ -204,6 +204,54 @@ export async function getAudioDuration(buffer: Buffer | string | Readable) {
 	return metadata.format.duration
 }
 
+/**
+  referenced from and modifying https://github.com/wppconnect-team/wa-js/blob/main/src/chat/functions/prepareAudioWaveform.ts
+ */
+export async function getAudioWaveform(buffer: Buffer | string | Readable, logger?: Logger) {
+	const AudioContext = require('web-audio-api').AudioContext
```

Comment:
Maybe, we can move it to inside of try catch

Reply:
Hmm, not really needed inside there, since this is a required dependency (It's only ES5 require that's why we use that here), it's 60KB in raw size, nothing to worry about.